ubiconfig/config_types/packages.py

```diff
@@ -13,7 +13,9 @@ def __init__(self, package, arches):
             self.arch = None
 
     def __repr__(self):
-        return self.package
+        if self.arch:
+            return "Package(name=%s, arch=%s)" % (self.name, self.arch)
```

Comment:
This doesn't exactly make sense since `Package.__init__` doesn't accept "name" or "arch" arguments.  So feeding this repr back into the console would crash.

I think repr ought to be one of:

- either it survives round trip eval(repr(x)) == x
- or it's using the surrounding angle brackets as in example `<Package: foobar>` to make it obvious it's some abridged representation of the object which can't be fed into eval

Actually the way you've designed this Package class right now, if you have an existing Package object, there's no way you can reconstruct the arguments passed to `__init__`.  If you really aim for eval(repr(x)) == x, it could get a bit weird or complicated here.  Then you may want to go for the second approach instead, use angle brackets and not aim for something which can be passed to eval.